gufe/network.py

```diff
@@ -0,0 +1,27 @@
+"""
```

Comment:
We had a minimal comment we were adding to all our `openfe` files, maybe the same thing can be done here? 

https://github.com/OpenFreeEnergy/openfe/blob/721abe4c4d1668cc6ab01e02bbcc22787156e92e/openfe/setup/ligand_network_planning.py#L1-L2

Reply:
Got it!